pytorch_blade/scripts/build_pytorch_blade.sh

```diff
@@ -46,12 +46,14 @@ function ci_build() {
     else
       python3 ../scripts/python/common_setup.py --cpu_only
     fi
+    TORCH_LIB=$(python -c 'import torch; import os; print(os.path.dirname(os.path.abspath(torch.__file__)) + "/lib/")') \
```

Comment:
Why should `TORCH_LIB` be added to `LD_LIBRARY_PATH`?

Reply:
Just keep with the older implementation, I find some code just like `bazel_build.py` uses this env, so I keep this env in this PR, and maybe polish them in the future.